src/ngx-slider/lib/slider.component.scss

```diff
@@ -1,4 +1,5 @@
-@import 'variables.scss';
+@use "sass:math";
```

Comment:
Why are you using sass math instead of native CSS variable computation? (I don't know anything about the difference yet).

Reply:
Sass math (and vars) runs at build time, while css math runs at runtime. 

So if the values dont change sass will be faster